src/main/java/org/myrobotlab/service/InMoov2.java

```diff
@@ -596,82 +486,55 @@ public String get(String key) {
   }
 
   public InMoov2Arm getArm(String side) {
-    if ("left".equals(side)) {
-      return leftArm;
-    } else if ("right".equals(side)) {
-      return rightArm;
-    } else {
-      log.error("can not get arm {}", side);
-    }
-    return null;
+    return (InMoov2Arm) getPeer(side + "Arm");
```

Comment:
there's no error checking that "side' value is valid here anymore.

Reply:
fixed